src/pages/dataTableUtilities/getColumns.js

```diff
@@ -225,13 +233,21 @@ const COLUMNS = () => ({
   },
 
   // STRING COLUMNS
-
+  [COLUMN_NAMES]: {
```

Comment:
Should this be?:

```suggestion
  [COLUMN_NAMES.PRESCRIBER]: {
```

Reply:
yup e900d57838e74655a9238c5be7e6b0eb25f7dadc